### PR TITLE
feature: add support for fiber

### PR DESCRIPTION
**Pull request by CodePrometheus**
[Fiber](https://github.com/gofiber/fiber) is a HTTP web framework.

I would appreciate it if the members of the community wouldn't mind taking a look at my code and providing some feedback.

### CODE REVIEW

**Comment by mrproliu** [comment]
@CodePrometheus Thanks for your contribution. The CI has been failure, please fix it first.

File: go.sum [hunk 1]
```diff
@@ -2,23 +2,14 @@ cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMT
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/OneOfOne/xxhash v1.2.2/go.mod h1:HSdplMjZKSmBqAxg5vPj2TmRDmfkzw+cTzAElWljhcU=
-github.com/andybalholm/brotli v1.0.5 h1:8uQZIdzKmjc/iuPu7O2ioW48L81FgatrcpfFmiq/cCs=
-github.com/andybalholm/brotli v1.0.5/go.mod h1:fO7iG3H7G2nSZ7m0zPUDn85XEX2GTukHGRSepvi9Eig=
```
Comment (wu-sheng) [inline go.sum:6]: @mrproliu Are these removing expected?

**Review by mrproliu — APPROVED** [review]
LGTM

File: go.sum (hunk 1 above)
Comment (CodePrometheus) [reply go.sum:6]: > @mrproliu Are these removing expected?

Judging from the final test results and my local regression tests, no relevant exceptions occurred.

**Comment by CodePrometheus** [comment]
> ID is registered. Note, we may need another PR on the UI side to represent the server on the topology.

@wu-sheng Teacher Wu, could you please explain in detail what else needs to be adjusted?

**Pull request merged**